src/classrooms/classrooms.controller.ts

```diff
@@ -102,4 +107,19 @@ export class ClassroomsController {
     const enrollments = await this.enrollmentsService.getStudentsForClassroom(classroomId);
     return this.enrollmentSerializer.serializeMany(enrollments);
   }
+
+  @Put(":classroomId/meet-link")
+  @Roles(EUserType.TEACHER)
+  async uploadMeetLink(
+    @Param("classroomId") classroomId: number,
+    @CurrentUser() user: { id: number },
+    @Body() updateMeetLinkDto: UploadMeetLinkDto,
+  ): Promise<ClassroomResponseDto> {
+    const classroom = await this.classroomsService.uploadMeetLink(
+      classroomId,
+      updateMeetLinkDto.meetLink,
+      user.id,
+    );
+    return this.classroomsSerializer.serialize(classroom);
```

Comment:
Do you need a new endpoint for this?
Can't you reuse the update classroom endpoint?

Reply:
Yeah I can, thing is that in the Todo app for marking a task completed I have used a separate endpoint without using update endpoint. So here I thought this would be same